src/routes/staking/staking-form.tsx

```diff
@@ -95,16 +98,29 @@ export const StakingForm = ({
         amount: removeDecimal(new BigNumber(amount), appState.decimals),
       },
     };
-    const undelegateInput: UndelegateSubmissionInput = {
+    const undelegateInput: UndelegateEndOfEpochSubmissionInput = {
       pubKey: pubkey,
       undelegateSubmission: {
         nodeId,
         amount: removeDecimal(new BigNumber(amount), appState.decimals),
         method: "METHOD_AT_END_OF_EPOCH",
       },
     };
+    const undelegateNowInput: UndelegateNowSubmission = {
+      pubKey: pubkey,
+      undelegateSubmission: {
+        nodeId,
+        amount: removeDecimal(new BigNumber(amount), appState.decimals),
+        method: "METHOD_NOW",
```

Comment:
Lets just keep only having two objects and nest a ternary in here to switch the method

Reply:
done